desktop/core/src/desktop/js/ko/components/ko.contextSelector.js

```diff
@@ -63,7 +63,7 @@ const TEMPLATE = `
     )}</span>
     <!-- /ko -->
 
-    <!-- ko if: availableNamespaces().length > 0 && !hideNamespaces -->
+    <!-- ko if: availableNamespaces().length > 0 && !hideNamespaces && !window.getLastKnownConfig().has_computes -->
```

Comment:
If I read this right it means we're only showing the namespace selection if we don't have computes. Which seems a bid odd to me. I think a better solution would be to instead change it to `ko if: availableNamespaces().length > 1 && ...` in this case.

Reply:
addressed